apps/web/src/app/(signing)/sign/[token]/signature-field.tsx

```diff
@@ -48,7 +48,7 @@ export const SignatureField = ({ field, recipient }: SignatureFieldProps) => {
 
   const [showSignatureModal, setShowSignatureModal] = useState(false);
   const [localSignature, setLocalSignature] = useState<string | null>(null);
-
+  const [isDocumentSigned, setIsDocumentSigned] = useState(false);
```

Comment:
I think `isDocumentSigned` should be updated to something like `isLocalSignatureSet` or something